src/frontend/src/pages/Register.tsx

```diff
@@ -8,86 +8,104 @@ import {
+      })
+         .then((response) => {
+            console.log(response);
+            setRes(response.data);
+         })
+         .catch((error) => {
+            if (error.response) {
+               console.log(error.response);
+               console.log(error.response.status);
+               console.log(error.response.headers);
+               setRes(error.response.data);
+            }
+         });
+   }
+   const navigate = useNavigate();
+
+   useEffect(() => {
+      if (res === `Register successful, welcome ${user}`) {
+         setTimeout(() => {
+            navigate('/login');
+         }, 2500);
+      }
+   }, [res, user, navigate]);
+
    return (
       <Grid>
          <Paper elevation={12} style={paperStyle}>
-            <PersonIcon fontSize="large" />
-            <h2>Register</h2>
-            <TextField
-               label="Username"
-               placeholder="Ex: user1"
-               onChange={(event) => setUser(event.target.value)}
-               fullWidth
-               required
-            />
-            <TextField
-               label="Email"
-               placeholder="Ex: user1@gmail.com"
-               onChange={(event) => setEmail(event.target.value)}
-               fullWidth
-               required
-            />
-            <TextField
-               label="Password"
-               placeholder="Ex: user1password!"
-               onChange={(event) => setPassword(event.target.value)}
-               type="password"
-               fullWidth
-               required
-            />
-            <TextField
-               label="Phone"
-               placeholder="Ex: (000)-000-0000"
-               onChange={(event) => setNumber(event.target.value)}
-               fullWidth
-               required
-            />
+            <Stack spacing={2}>
+               {/* A paper like UI with fields for register*/}
+               <Grid container spacing={2}>
+                  <Grid item xs={2}>
+                     <PersonIcon fontSize="large" />
+                  </Grid>
+                  <Grid item xs>
+                     <h2>Register</h2>
+                  </Grid>
+               </Grid>
+               <TextField
+                  label="Username"
+                  placeholder="Ex: user1"
+                  onChange={(event) => setUser(event.target.value)}
+                  fullWidth
+                  required
+               />
+               <TextField
+                  label="Email"
+                  placeholder="Ex: user1@gmail.com"
+                  onChange={(event) => setEmail(event.target.value)}
+                  fullWidth
+                  required
+               />
+               <TextField
+                  label="Password"
+                  placeholder="Ex: user1password!"
+                  onChange={(event) => setPassword(event.target.value)}
+                  type="password"
+                  fullWidth
+                  required
+               />
```

Comment:
Can we also have a note like "password must be 8 characters"?

Reply:
Noted